### DISCUSSION TITLE
HyperMorph code

### DISCUSSION THREAD

**Discussion by xiaogengen-007** [voxelmorph/voxelmorph#415] [Q&A] [answered]
Hi,

I really like the idea of HyperMorph and would like to try it out for our hyperparameter tuning? I didn't find any related code under the current voxelmorph repo. Are there any future plans for HyperMorph training to be open-source?

Thank you for the great work!

**Comment by adalca** [comment, marked as answer]
The code should be up as of the fall of 2021!

The script for training is here: https://github.com/voxelmorph/voxelmorph/blob/dev/scripts/tf/train_hypermorph.py
which uses the network from vxm/tf/networks

@ahoopes can comment more.

**Reply by xiaogengen-007** [reply]
This is great! I only looked under the PyTorch folder. Thank you!
